### DISCUSSION TITLE
Links with anchors point to the top of the page in Chrome

### DISCUSSION THREAD

**Discussion by jan-tosovsky-cz** [mdn/mdn-community#351] [Content]
When sharing the link with an anchor, e.g. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#improving_scrolling_performance_with_passive_listeners it is scrolled properly in Firefox, but that anchor is ignored in Chrome.
As anchors in other sites work in Chrome I assume there is some JS manipulation on the MDN site which breaks this.
I am not sure this is the proper place to report this. If not, please point me to the better one. Thanks!

**Comment by teoli2003** [comment]
Thank you for your report.

This is handled in this issue: https://github.com/mdn/yari/issues/8049

**Comment by bsmth** [comment]
Closing as this is being tracked in the linked issue. Thank you!
